src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -246,6 +246,8 @@
   &.pf-m-secondary {
     --#{$menu-toggle}--Color: var(--#{$menu-toggle}--m-secondary--Color);
     --#{$menu-toggle}--m-expanded--Color: var(--#{$menu-toggle}--m-secondary--m-expanded--Color);
+    
+    background-color: var(--#{$menu-toggle}--m-secondary--BackgroundColor);
```

Comment:
Our model has been to assign the modifier variable to the base variable for the property, although we've had a little bit of discuss around getting away from this. For now I think I'd prefer to stay consistent with the model.

Reply:
Sounds good, I've updated the pr. This required the `hover` `focus` `active` and `expanded` variables to be explicitly set.  Let me know if this isn't what's preferred.

Reply:
One item I noticed after pushing my changes is github is recognizing a [newline addition at the end of file](https://github.com/patternfly/patternfly/pull/7140/files#diff-62015d4798b534885d858ec2db1ee853e67d6eedcfde5142cd4f4364e88cbd52R639). 

My VS code settings for `Insert Final Newline` is not checked so I'm not sure why that's being added. Any ideas? 

@mcoker would you know? Do the PF devs use a shared [EditorConfig](https://editorconfig.org/) file, that I should use, for consistency?

Reply:
We don't have a shared editor config, but we do put an EOF at the end of files by default, unless there is a specific reason not to. A reason not to might be something like you want to control/remove the whitespace in the content of some element, and the content comes from a few `.hbs` files being included in one another, so you need to strip the `.hbs` files of their whitespace. But that case is pretty few and far between.

It's not a big deal to omit the newline, but if possible you should include it just for consistency's sake. It also can make the markup (or whatever the file is) more readable if there is a new line at the end of it. Otherwise you may get something like this:

```
// foo.hbs
<div>
  <span>foo</span>
</div> // no newline at EOF here
```

```
// bar.hbs
<div>
  <span>bar</span>
</div>
```

```
// foobar.hbs - the rendered HTML from the 2 files above included in some component example
<div>
  <span>foo</span>
</div><div> // these are on the same line from the lack of a newline at EOF in foo.hbs
  <span>bar</span>
</div>
```